### PR TITLE
update index

### LINKED ISSUE #10: Update home page content
- [x] #45
- [x] check that the stylesheet is linked properly in the HTML file
- [x] add website title to nav and footer 
- [x] determine hero section content - bg image, headline, and subtitle
- [x] determine page content - _maybe some kind of preview or image directing the user to each section (Spotify and Ticketmaster)? May be redundant but would keep the page from looking too sparse if we can't think of anything else_

### PR DESCRIPTION
**Pull request by gabster1313**
adding images for spotify and Ticketmaster + link

### CODE REVIEW

File: images/Tickets.png
Comment (Cassie-Andrews) [inline images/Tickets.png:1]: cuuute!!

File: spotify.html
Comment (Cassie-Andrews) [inline spotify.html:1]: For some reason, this update shows spotify.html, ticket.html, and ticket.js as deleted. Is this how it looks on your end too?

File: images/Tickets.png
Comment (gabster1313) [reply images/Tickets.png:1]: Hi! Can you push this so i can push the one you submitted so the code is accurate for some reason this one didn’t get pushed when you reviewed it

**Review by Cassie-Andrews — APPROVED** [review]

File: spotify.html
Comment (gabster1313) [reply spotify.html:1]: Wtf how weird I’m gonna push through the ones you just sent and add my updates to those and re do it

**Pull request closed without merging**